.bazelrc

```diff
@@ -70,12 +70,14 @@ build:rpi3 --host_crosstool_top=@bazel_tools//tools/cpp:toolchain
 build:rpi3 --crosstool_top=//tools/arm_compiler/linaro-gcc72-armeabi:toolchain
 build:rpi3 --cpu=armv7 --define=target_system=rpi3
 build:rpi3 --copt=-march=armv7-a --copt=-mtune=cortex-a53 --copt=-mfloat-abi=hard --copt=-mfpu=neon-fp-armv8 --copt=-DRASPBERRY_PI --copt=-D_GLIBCXX_USE_CXX11_ABI=0 --copt=-std=gnu99 --copt=-mno-unaligned-access
+build:rpi3 --define=tensorflow_mkldnn_contraction_kernel=0
```

Comment:
Is there any reason we only need that for ARM build ? Is it the default on other platforms and we just need to enforce here because we cross-compile our way ?

Reply:
The default changed to *enable* MKL contraction kernel on other platforms, but it doesn't work at all for the ARM targets, as is typical in Intel stuff.